server/ai_mediaserver.go

```diff
@@ -471,7 +471,8 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 		// if auth webhook returns pipeline config these will be replaced
 		pipeline := qp.Get("pipeline")
 		rawParams := qp.Get("params")
-		var streamID, pipelineID string
+		streamID := qp.Get("streamId")
```

Comment:
This is a not related change, but it's needed to test everything on a local dev machine. It allows specifying the streamID with the query params.